README.md

```diff
@@ -41,22 +29,32 @@ One of the very common use cases of WebXR is 180 and/or 360 photos or videos, bo
 Another example where composition layers are going to shine is displaying text or high resolution textures in the virtual world. Since composition layers allow to sample source texture at full resolution without downsampling to the resolution of the eye buffers (which is usually much lower than the physical resolution of the device's screen(s)) the readability of the text is going to be significantly improved.
 
 
-### Implementation overview
+## Goals of the proposal
+The purpose of this document is to unlock the ability for User Agents to experiment with multiple layers. 
+* Introduce a concept of layers into WebXR spec and the way how layers should be provided from user's code to Browser;
```

Comment:
The concept of layers already exists in the WebXR spec.  Did you mean the concept of multiple layers?

Reply:
yes